spec/factories/patients.rb

```diff
@@ -15,21 +15,29 @@
     date_of_birth { Date.today if has_date_of_birth? }
     age { rand(18..100) unless has_date_of_birth? }
     age_updated_at { Time.now - rand(10).days if age.present? }
-    created_at { Time.now }
-    updated_at { Time.now }
-    updated_on_server_at { Time.now }
+    device_created_at { Time.now }
+    device_updated_at { Time.now }
     association :address, strategy: :build
     phone_numbers { build_list(:patient_phone_number, rand(1..3), patient_id: id) }
   end
 end
 
+def with_payload_keys(attributes)
```

Comment:
Nevermind, found it! This code duplication smells wrong. Shouldn't this test the model code itself?